.env.example

```
@@ -19,6 +19,19 @@ DATABASE_WORKER_ROLE=openschool_worker
 # Production rollback disables the vertical slice; it never restores owner access.
 OPENSCHOOL_STUDENT_SLICE_MODE=forced_rls
 
+# Invitation links are stored only as AES-256-GCM ciphertext. Keep historical
+# keys in the JSON keyring until every row encrypted with them is terminal.
+INVITATION_TOKEN_ENCRYPTION_KEY_ID=local-v1
+INVITATION_TOKEN_ENCRYPTION_KEYS={"local-v1":"AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"}
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)